u-boot/cpu/mips/ar7240/qca_dram.c

```diff
@@ -624,6 +629,11 @@ static inline void qca_dram_set_ddr_cfg2(u32 mem_cas,
 
 	/* Gate open latency = 2 * MEM_CAS */
 	tmp = 2 * mem_cas;
+#if (SOC_TYPE & QCA_AR933X_SOC)
```

Comment:
I'm really not sure about this one. In datasheet I have:

Register DDR_CONFIG2:
- GATE_OPEN_LATENCY: Memory CAS_LATENCY * 2

Register DDR_CONFIG:
- CAS_LATENCY: DRAM CAS latency parameter (first 3 bits) rounded up in memory core clock cycles. Used by the hardware to estimate the internal DDR clock latency of a read. It should be >= GATE_OPEN_LATENCY as specified in the DDR_CONFIG2 register. The value of this register should be Memory CAS_LATENCY * 2 or CAS_LATENCY * 2 + 1|2|3.

Have you tried without touching DDR_CONFIG2 and only change value in DDR_CONFIG?